appdatabase/database.go

```diff
@@ -61,6 +63,7 @@ func doMigration(db *sql.DB) error {
 
 	postSteps := []*sqlite.PostStep{
 		{Version: 1662365868, CustomMigration: FixMissingKeyUIDForAccounts},
+		{Version: 1662365868, CustomMigration: EnableLightClientForMobileV1ByDefault},
```

Comment:
I don't have experience with the details of sqlite's `Migrate` function, but it does sort the custom steps slice based on `Version`, so slightly more correct is to increment the new step by one, not use the identical version to be extra precise one should happen after the other.